src/components/core/NumberInput/NumberInput.tsx

```diff
@@ -11,7 +11,6 @@ import {
   PrimitiveNumberInputScrubber,
```

Comment:
Same as the `Input` component, this component did not correctly have all styles applied until this refactor, so updating it would be ideal (https://trello.com/c/ZNF87PAY/321-clean-up-input-and-numberinput-component)

Reply:
Add `RadioGroup` to this list please

Reply:
`RadioGroup` also broken before this PR, but will fix in the upcoming `Input`/`NumberInput` patches